apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -906,6 +925,42 @@ class ScriptEditor extends React.Component {
               }
             />
           </label>
+          <div>
+            <Button
+              name="launch_plc_course"
+              text="Launch PLC Course"
+              onClick={this.launchPlcCourse}
+              color={Button.ButtonColor.purple}
+              disabled={
+                !(this.state.title && this.state.professionalLearningCourse)
+              }
```

Comment:
* also disable button if `this.state.plcCourseLaunchStatus === 'Course Launched'` ?

* also disable button if the course was previously launched?